appinventor/appengine/war/blocklyframe.html

```diff
@@ -10,6 +11,7 @@
     <style>
       html, body {
       background-color: #fff;
+      height: 100%;
```

Comment:
When I change the doctype, the blocks editor does not fill its parent element completely [quirks mode vs standard mode]. This change ensures that the blocks editor fills the whole height.